### DISCUSSION TITLE
Getting Lerna error (unexpected token) during build

### DISCUSSION THREAD

**Discussion by 1988vignesh** [lerna/lerna#3464] [Q&A]
I was using learn 4 ealier with ^ symbols  like "lerna : ^4.0.0" in my pacakge.json so it will always look for latest version of lerna
when we build the application recenlty we getting unexpected token error as show below.
below is the error,
node_modules/lerna/node_modules/@lerna/pakage-graph/index.js:79
if(deoNode?.version) {

syntaxError: unexpected token '.'

help is much appriciated

**Comment by ghiscoding** [comment]
A simple search in the issues seems to provide you with this possible answer
https://github.com/lerna/lerna/issues/3368
